source/index.html.haml

```diff
@@ -85,6 +85,10 @@
 .audio-container
   .audio
     %audio(src="paragliding.mp3" preload="auto" autoplay controls)
+  .hlwr
+    = link_to  "<~", "http://hotlinewebring.club/KeepRubyWeird/previous"
```

Comment:
It looks like there are two spaces after `link_to` (instead of one) on the three lines added to this file.